tests/phpunit/tests/site-health.php

```diff
@@ -59,6 +59,21 @@ public function test_cron_health_checks( $times, $expected_status, $expected_lab
 		$this->assertSame( $expected_missed, $wp_site_health->has_missed_cron() );
 	}
 
+	/**
+	 * Ensure Site Health reports correctly WebP support.
+	 */
+	public function test_webp_support_check() {
+		if ( ! wp_image_editor_supports( array( 'mime_type' => 'image/webp' ) ) ) {
+			$this->markTestSkipped( 'WebP image format is not supported on this system.' );
```

Comment:
This wording is a bit odd, but I see other doc block comments in the same file use the exact same form.